roosevelt.js

```diff
@@ -104,6 +156,21 @@ module.exports = function(params) {
       };
   
   function startServer() {
+    var lock = {},
```

Comment:
I have a question about the lock. Did you implement it so that the `onServerStart` event fired only once in the case that HTTPS was enabled and `httpsOnly` was disabled (two servers)?